def log_msg_with_error(msg: str, context):
    print(msg)
    print(context)


def log_msg(msg: str):
    print(msg)
